src/metabase/util.cljc

```diff
@@ -202,22 +204,26 @@
 (def ^{:arglists '([x])} ->kebab-case-en
   "Like [[camel-snake-kebab.core/->kebab-case]], but always uses English for lower-casing, supports keywords with
   namespaces, and returns `nil` when passed `nil` (rather than throwing an exception)."
-  (wrap-csk-conversion-fn-to-handle-nil-and-namespaced-keywords ->kebab-case-en*))
+  (wrap-csk-conversion-fn-to-handle-nil-and-namespaced-keywords
+   (memoize/lru ->kebab-case-en* :lru/threshold 256)))
```

Comment:
[generic-split](https://github.com/clj-commons/camel-snake-kebab/blob/ac08444c94aca4cba25d86f3b3faf36596809380/src/camel_snake_kebab/internals/string_separator.cljc#L42) in the camel-snake-kebab library turned out to be too expensive.